assets/_scss/components/_forms.scss

```diff
@@ -23,7 +23,7 @@ form {
     margin-bottom: 1.5em;
   }
 
-  input[name="password"] {
+  input[name="password"], input[name="confirmPassword"] {
```

Comment:
First off, does this work with just the attribute selector?

```
[name="password", [name="confirmPassword"]
```

If it doesn't work, then having the tag in there as well is fine, but was hoping we could move to having just attribute selectors across the site to fix specificity problems.

Also, does adding bottom margin to these two password inputs work in different configurations of form elements? Such as when you could have password inputs before another form element?

Generally, I think it would be more successful if all these inputs used top margins and adjacent sibling selector, so we could do something more like this:

```
form > * + *
```

but we're a while away from that so if the two things I bring up are looked at, I'm OK with this change in the mean time.